README.md

```diff
@@ -38,7 +38,7 @@ See what [people are saying about Rich](https://www.willmcgugan.com/blog/pages/p
 
 ## Compatibility
 
-Rich works with Linux, OSX, and Windows. True color / emoji works with new Windows Terminal, classic terminal is limited to 16 colors. Rich requires Python 3.7 or later.
+Rich works with Linux, OSX, and Windows. True color and emoji work on the new Windows Terminal, classic terminal is limited to 16 colors. Rich requires Python 3.7 or later.
```

Comment:
Instead of ", classic terminal ...", it would be better to either use "; classic terminal ..." or ", whereas classic terminal ..."